Source/Extensions/Blazorise.TreeView/Internal/_TreeViewNodeContent.razor

```diff
@@ -0,0 +1,13 @@
+﻿@typeparam TNode
+@inherits BaseComponent
+<div class="b-tree-view-node-title">
+    @if ( ParentTreeViewState.SelectionMode == TreeViewSelectionMode.Multiple )
+    {
+        <div class="b-tree-view-node-check">
+            <Check TValue="bool" Checked="@Checked" CheckedChanged="@OnCheckedChanged" />
```

Comment:
Did you have any problems in syncing stuff up? Check selection vs regular selection? Tested all providers? I believe Datagrid wasn't easy to get right at the time. But I can't recall the exact problems we had.

Reply:
Yeah, I have checked with all providers, and it seems to work with the current code structure. I must admit it wasn't easy to do, though.